### PR TITLE
Fix building binaries commands so they pick up the GOOS and GOARCH vars

### PR DESCRIPTION
**Pull request by vreynolds**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -19,27 +19,31 @@ commands:
         default: "amd64"
     steps:
       - run: |
+          cd cmd/honeyalb && \
           GOOS=<< parameters.os >> \
           GOARCH=<< parameters.arch >> \
-          cd cmd/honeyalb && go build -ldflags "-X main.BuildID=${CIRCLE_TAG}" \
+          go build -ldflags "-X main.BuildID=${CIRCLE_TAG}" \
           -o $GOPATH/bin/honeyalb-<< parameters.os >>-<< parameters.arch >> \
           .
```
Comment (robbkidd) [inline .circleci/config.yml:27]: Would rubbing moar Circle on these run steps help avoid multi-line shell ordering trouble?

```yaml
        run:
          working_directory: cmd/honeyalb
          environment:
            GOOS=<< parameters.os >>
            GOARCH=<< parameters.arch >>
          command: | 
            go build -ldflags "-X main.BuildID=${CIRCLE_TAG}" \
            -o $GOPATH/bin/honeyalb-<< parameters.os >>-<< parameters.arch >> \
            .
```

**Review by robbkidd — APPROVED** [review]
Seems an improvement to me. 👍 

Left a small comment about maybe being able to break the multiline command up into component parts of a CircleCI [run](https://circleci.com/docs/2.0/configuration-reference/#run) step.

**Comment by vreynolds** [comment]
I like your suggestion @robbkidd, PR updated

**Review by robbkidd — APPROVED** [review]
Mmm. Purdy.

**Pull request merged**